composer.json

```diff
@@ -31,14 +31,7 @@
     "heroku/heroku-buildpack-php": "*"},
     "scripts": {
         "compile": [
-            "cp config.ini.php vendor/piwik/piwik/config/config.ini.php",
-            "sed -i s/#DB_HOST/$DB_HOST/ vendor/piwik/piwik/config/config.ini.php",
-            "sed -i s/#DB_USERNAME/$DB_USERNAME/ vendor/piwik/piwik/config/config.ini.php",
-            "sed -i s/#DB_PASSWORD/$DB_PASSWORD/ vendor/piwik/piwik/config/config.ini.php",
-            "sed -i s/#DB_NAME/$DB_NAME/ vendor/piwik/piwik/config/config.ini.php",
-            "sed -i s/#DB_PORT/$DB_PORT/ vendor/piwik/piwik/config/config.ini.php",
-            "sed -i s/#DB_PREFIX/$DB_PREFIX/ vendor/piwik/piwik/config/config.ini.php",
-            "sed -i s/#SECRET_TOKEN/$SECRET_TOKEN/ vendor/piwik/piwik/config/config.ini.php"
+            "cp img.php vendor/piwik/piwik/img.php"
```

Comment:
Oops!  I didn't catch this, it belongs in a different branch.  I can rebase this again.